src/alda/AldaServer.java

```diff
@@ -441,6 +441,23 @@ public AldaResponse playStatus(byte[] workerAddress, String jobId)
     return req.send();
   }
 
+  public void stop() {
+    AldaRequest req = new AldaRequest(host, port);
+    req.command = "stop-playback";
+
+    try {
+      AldaResponse res = req.send();
+
+      if (res.success) {
+        msg(res.body);
+      } else {
+        error(res.body);
+      }
+    } catch (NoResponseException e) {
+      serverDown();
```

Comment:
kind of a nitpick, but could this method be renamed to `printServerDown` or something like that? Right now the name slightly implies that this shuts down the server... :P